geopyspark/geotrellis/histogram.py

```diff
@@ -121,6 +133,19 @@ def bucket_count(self):
 
         return self.scala_histogram.bucketCount()
 
+    def bin_counts(self):
+        """Returns a list of tuples where the key is the bin label value and the
+        value is the label's respective count.
+
+        Returns:
+            [(int, int)] or [(float, int)]
+        """
+
+        labels = self.values()
```

Comment:
`bin_counts` is executed in Python because for some reason, Py4J cannot find the `binCounts()` Scala method.

```
E                   py4j.protocol.Py4JError: An error occurred while calling o47.binCounts. Trace:
E                   py4j.Py4JException: Method binCounts([]) does not exist
E                   	at py4j.reflection.ReflectionEngine.getMethod(ReflectionEngine.java:318)
E                   	at py4j.reflection.ReflectionEngine.getMethod(ReflectionEngine.java:326)
E                   	at py4j.Gateway.invoke(Gateway.java:272)
E                   	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
E                   	at py4j.commands.CallCommand.execute(CallCommand.java:79)
E                   	at py4j.GatewayConnection.run(GatewayConnection.java:214)
E                   	at java.lang.Thread.run(Thread.java:745)
```